feature-detail/src/androidMain/kotlin/detail/data/DetailStreamRepository.kt

```diff
@@ -1,12 +1,12 @@
-package com.codandotv.streamplayerapp.feature_list_streams.detail.data
+package detail.data
 
 import com.codandotv.streamplayerapp.core_local_storage.data.dao.FavoriteDao
 import com.codandotv.streamplayerapp.core_networking.handleError.toFlow
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.DetailStream
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.VideoStream
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.toDetailStream
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.toDetailStreamLocal
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.toVideoStreams
+import detail.domain.DetailStream
```

Comment:
The same point I said before, I think it is something going wrong with our name space for this module